modules/migration-mtc-release-notes-1-7-06.adoc

```diff
@@ -0,0 +1,30 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-06_{context}"]
+= {mtc-full} 1.7.6 release notes
+
+[id="new-features-1-7-6_{context}"]
+== New features
+
+.Implement proposed changes for DVM support with PSA in Red Hat OpenShift Container Platform 4.12
+With the incoming enforcement of Pod Security Admission (PSA) in {OCP} 4.12, whereby the default pod would run with a `restricted` profile. This `restricted` profile would mean workloads to migrate would be in violation of this policy and no longer work as of now. The above enhancement outlines the changes that will be required to remain compatible with OCP 4.12. (link:https://issues.redhat.com/browse/MIG-1240[*MIG-1240*])
```

Comment:
```suggestion
With the incoming enforcement of Pod Security Admission (PSA) in {OCP} 4.12 the default pod would run with a `restricted` profile. This `restricted` profile would mean workloads to migrate would be in violation of this policy and no longer work as of now. The following enhancement outlines the changes that will be required to remain compatible with OCP 4.12. (link:https://issues.redhat.com/browse/MIG-1240[*MIG-1240*])
```
This needs to be reworded slightly to avoid a run-on sentence. I've made a suggestion but you can feel free to adjust it.